src/text/text3.asm

```diff
@@ -619,7 +619,7 @@ Text02f5: ; 3d2e6 (f:52e6)
 	line "very rare Pokémon cards!"
 	done
 
-Text02f6: ; 3d31c (f:531c)
+GlossaryText: ; 3d31c (f:531c)
```

Comment:
Considering that this text is short and that ``GlossaryText`` would conflict with the text at e:7cdb (which is, precisely, "Glossary", I would simply rename this to ``PokemonCardGlossaryText``.